src/components/PeopleTable.tsx

```diff
@@ -10,44 +35,62 @@ export const PeopleTable = () => {
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Name
```

Comment:
You can create array of CONSTANTS with sorting fields values and map it here. You will have shorter code